logos/logos-workernode/tests/test_sharded_checkpoint.py

```
@@ -0,0 +1,162 @@
+def _lane(tp: int, tmp_path: Path) -> LaneConfig:
+    return LaneConfig(
+        model="org/Model-A",
+        vllm=True,
+        gpu_devices="0,1",
+        vllm_config=VllmConfig(tensor_parallel_size=tp, enable_sleep_mode=True),
+    )
+
+
+def test_build_cmd_uses_sharded_dir(tmp_path: Path, monkeypatch) -> None:
+    handle = VllmProcessHandle("lane-x", 19010, OllamaConfig(), VllmEngineConfig())
+    monkeypatch.setattr(handle, "_resolve_vllm_binary", lambda _c: ["/tmp/vllm"])
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/vllm"

(S108)